### PR TITLE
Added workshop Designers' interview process

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Added md file
- Filled in tags and added description

### CODE REVIEW

File: _calendar/2022-03-22-design-hiring-workshop.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: 'Understanding the designers' interview process'
+image: '/assets/images/calendar/2022-03-22-design-hiring-workshop/logo.png'
+location: Berlin
+url:
```
Comment (marcoow) [inline _calendar/2022-03-22-design-hiring-workshop.md]:
```suggestion
```

**Comment by emmasofiah2o** [comment]
If you click on "visit website" it takes you back to the calendar page

**Comment by marcoow** [comment]
let's just link to Linda's LI post

**Comment by marcoow** [comment]
@emmasofiah2o please re-request reviews after making changes – I missed this and now it's too late as the event was last tuesday :)

**Pull request closed without merging**